### PR TITLE
LPS-167920 Test Scenarios Coverage | Backend

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java [hunk 1]
```diff
@@ -269,6 +269,42 @@ public void testAddKBArticlesMarkdownWithSingleApproverWorkflow()
 		importMarkdownArticles();
 	}
 
+	@Test
+	public void testAddKBArticleUpdateExpirationDateUpdateStatus()
```
Comment (marcogalluzzi) [inline modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java:273]: I would rename it to something like `testExpireKBArticleUpdatesStatus()`

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]
Test looks OK. Just an optional method renaming.

File: modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java]: I will push a name change to `testUpdateKBArticleExpirationDateUpdatesStatus`

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java (hunk 1 above)
Comment (marcogalluzzi) [reply modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/service/test/KBArticleLocalServiceTest.java]: New name is better. Thanks

**Review by marcogalluzzi — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
ci:forward

**Pull request closed without merging**